docs/en/operation/Configuration.md

```diff
@@ -125,14 +127,13 @@ $ cp conf/alluxio-env.sh.template conf/alluxio-env.sh
 with the cluster-wide configuration values retrieved from Alluxio masters.
 
 When different client applications (Alluxio Shell CLI, Spark jobs, MapReduce jobs)
-or Alluxio workers connect to an Alluxio master,
-they will initialize their own Alluxio configuration properties with the default values
-supplied by the masters based on the master-side `${ALLUXIO_HOME}/conf/alluxio-site.properties` files.
-As a result, cluster admins can set client-side settings (e.g., `alluxio.user.*`),
-or network transport settings (e.g., `alluxio.security.authentication.type`),
-or worker settings (e.g., `alluxio.worker.*`)
-in `${ALLUXIO_HOME}/conf/alluxio-site.properties` on all the masters,
-which will be distributed and become cluster-wide default values when clients and workers connect.
+or Alluxio workers connect to an Alluxio master, they will initialize their own Alluxio
+configuration properties with the default values supplied by the masters based on the master-side
+`${ALLUXIO_HOME}/conf/alluxio-site.properties` files.
+As a result, cluster admins can set default client-side settings (e.g., `alluxio.user.*`), or
+network transport settings (e.g., `alluxio.security.authentication.type`) in
+`${ALLUXIO_HOME}/conf/alluxio-site.properties` on all the masters, which will be distributed and
+become cluster-wide default values when clients and workers connect.
```

Comment:
remove workers from this line

Reply:
AFAIK alluxio workers contain an Alluxio client and will get cluster defaults for those from the master, so this is still correct.

ex:
https://github.com/Alluxio/alluxio/blob/master/core/server/worker/src/main/java/alluxio/worker/block/BlockMasterSync.java#L107-L108